doc/source/user/feature-classification/feature-matrix-listener.ini

```diff
@@ -0,0 +1,331 @@
+cli=openstack loadbalancer listener create [--insert-headers X-SSL-Client-Not-Aftr=true] <loadbalancer>
+driver.elb=missing
+driver.vlb=missing
+
+[operation.name]
+title=name
+status=optional
+notes=The name of the load balancer listener. Provided by the Octavia API service.
+cli=openstack loadbalancer listener create [--name <name>] <loadbalancer>
+driver.elb=complete
+driver.vlb=complete
+
+[operation.protocol.HTTP]
+title=protocol - HTTP
+status=optional
+notes=HTTP protocol support for the listener.
+cli=openstack loadbalancer listener create --protocol HTTP <loadbalancer>
+driver.elb=complete
+driver.vlb=missing
+
+[operation.protocol.HTTPS]
+title=protocol - HTTPS
+status=optional
+notes=HTTPS protocol support for the listener.
+cli=openstack loadbalancer listener create --protocol HTTPS <loadbalancer>
+driver.elb=partial
+driver.vlb=partial
+driver-notes.elb=Implemented internally as TCP.
+driver-notes.vlb=Implemented internally as TCP.
+
+[operation.protocol.TCP]
+title=protocol - TCP
+status=optional
+notes=TCP protocol support for the listener.
+cli=openstack loadbalancer listener create --protocol TCP <loadbalancer>
+driver.elb=complete
+driver.vlb=complete
+
+[operation.protocol.TERMINATED_HTTPS]
+title=protocol - TERMINATED_HTTPS
+status=optional
+notes=Terminated HTTPS protocol support for the listener.
+cli=openstack loadbalancer listener create --protocol TERMINATED_HTTPS <loadbalancer>
+driver.elb=complete
+driver.vlb=complete
+
+[operation.protocol.UDP]
+title=protocol - UDP
+status=optional
+notes=UDP protocol support for the listener.
+cli=openstack loadbalancer listener create --protocol UDP <loadbalancer>
+driver.elb=complete
+driver.vlb=complete
+
+[operation.protocol.SCTP]
+title=protocol - SCTP
+status=optional
+notes=SCTP protocol support for the listener.
+cli=openstack loadbalancer listener create --protocol SCTP <loadbalancer>
+driver.elb=missing
+driver.vlb=missing
+
+[operation.protocol_port]
+title=protocol_port
+status=mandatory
+notes=The protocol port number for the listener.
+cli=openstack loadbalancer listener create --protocol-port <port> <loadbalancer>
+driver.elb=complete
+driver.vlb=complete
+
+[operation.sni_container_refs]
+title=sni_container_refs
+status=optional
+notes=A list of URIs to the key manager service secrets containing PKCS12 format certificate/key bundles for TERMINATED_TLS listeners.
+cli=openstack loadbalancer listener create [--sni-container-refs [<container_ref> [<container_ref> ...]]] <loadbalancer>
+driver.elb=complete
+driver.vlb=complete
+
+[operation.stats]
+title=Listener statistics
+status=mandatory
+notes=The ability to show statistics for a listener.
+cli=openstack loadbalancer listener stats show <listener>
+driver.elb=missing
+driver.vlb=missing
+
+[operation.tags]
+title=tags
+status=optional
+notes=The tags for the load balancer listener. Provided by the Octavia API service.
+cli=openstack loadbalancer listener create [--tags <tag>] <loadbalancer>
+driver.elb=complete
```

Comment:
tags are not supported in elbv2 according to ELB API doc

Reply:
they are (https://docs.otc.t-systems.com/api/elb/elb_zq_bq_0001.html). The elb doc is designed badly